### DISCUSSION TITLE
Move Roboto glyph vendor to a more accessible place?

### DISCUSSION THREAD

**Discussion by Wheemix** [chromiecraft/chromiecraft#6049] [Suggestions]
Especially now that we live in Outlands, I feel like the trip to Tanaris is unnecessarily tedious. Cant even use your mage alt to teleport to Theramore etc because you can only buy glyphs for the class you're on

**Comment by Zelfie** [comment]
There's a blood elf woman in the lower city bar that can teleport you to the caverns of time.
Might require a bit of rep
